<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>repkassa</class>
 <widget class="QWidget" name="repkassa">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>432</width>
    <height>126</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>8</x>
     <y>4</y>
     <width>421</width>
     <height>85</height>
    </rect>
   </property>
   <property name="title">
    <string>Параметры отчета</string>
   </property>
   <widget class="QDateLineEdit" name="date_start">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>20</y>
      <width>101</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>71</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Дата начала:</string>
    </property>
   </widget>
   <widget class="QDateLineEdit" name="date_end">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>20</y>
      <width>101</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>20</y>
      <width>91</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Дата окончания:</string>
    </property>
   </widget>
   <widget class="QSelLineEdit" name="Sotrudnik">
    <property name="geometry">
     <rect>
      <x>8</x>
      <y>58</y>
      <width>281</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>8</x>
      <y>44</y>
      <width>46</width>
      <height>11</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Мастер:</string>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="start_report">
   <property name="geometry">
    <rect>
     <x>268</x>
     <y>92</y>
     <width>161</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Сформировать</string>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QDateLineEdit</class>
   <extends>QWidget</extends>
   <header>qdatelineedit.h</header>
  </customwidget>
  <customwidget>
   <class>QSelLineEdit</class>
   <extends>QLineEdit</extends>
   <header>qsellineedit.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
